charts/oidc-proxy/ci/testing-values.yaml

```diff
@@ -0,0 +1,19 @@
+fullnameOverride: test-oidc-proxy
+env:
+  PROXY_URL: http://test-datarepo-ui.integration-temp:8080/
+  PROXY_URL2: http://test-datarepo-api.integration-temp:8080/api
+  PROXY_URL3: http://test-datarepo-api.integration-temp:8080/register
+  LOG_LEVEL: debug
+  SERVER_NAME: jade.datarepo-integration.broadinstitute.org
+  REMOTE_USER_CLAIM: sub
+  ENABLE_STACKDRIVER: yes
```

Comment:
huh interesting..very meta...did the lintintg PR catch a linting issue with itself?

Reply:
not my docker image but it wants yes lol